lensless/utils/dataset.py

```diff
@@ -1508,6 +1514,26 @@ def __init__(
             )
             self.simulator = simulator
 
+        if bg_fp is not None and os.path.isfile(bg_fp):
+            assert (
+                bg_snr_range is not None
+            ), "Since a background path was provided, the SNR range should not be empty"
+            bg = load_image(
+                bg_fp,
+                shape=lensless.shape,
+                return_float=True,
+                flip=rotate,
+            )
+            self.bg = torch.from_numpy(bg)
```

Comment:
can you rename this to `self.bg_sim`? and change in other places. So that we can distinguish from measured background when we get to implementing that